src/quo/components/drawers/bottom_actions/view.cljs

```diff
@@ -2,55 +2,105 @@
   (:require
     [quo.components.buttons.button.view :as button]
     [quo.components.drawers.bottom-actions.style :as style]
+    [quo.components.icon :as icon]
     [quo.components.markdown.text :as text]
+    [quo.components.tags.context-tag.view :as context-tag]
+    [quo.foundations.colors :as colors]
     [quo.theme :as quo.theme]
-    [react-native.core :as rn]))
+    [react-native.blur :as blur]
+    [react-native.core :as rn]
+    [schema.core :as schema]
+    [utils.i18n :as i18n]))
 
-(def default-props
-  {:button-one-type     :primary
-   :button-two-type     :grey
-   :customization-color :blue})
+(def ?schema
+  [:=>
+   [:catn
+    [:props
+     [:map {:closed true}
+      [:actions [:maybe [:enum :one-action :two-actions]]]
+      [:description {:optional true} [:maybe [:enum :top :bottom :top-error]]]
+      [:description-text {:optional true} [:maybe :string]]
+      [:error-message {:optional true} [:maybe :string]]
+      [:role {:optional true} [:maybe [:enum :admin :member :token-master :owner]]]
+      [:button-one-label {:optional true} [:maybe :string]]
+      [:button-two-label {:optional true} [:maybe :string]]
+      [:button-one-props {:optional true} [:maybe :map]]
+      [:button-two-props {:optional true} [:maybe :map]]
+      [:theme :schema.common/theme]
+      [:scroll? {:optional true} [:maybe :boolean]]
+      [:blur? {:optional true} [:maybe :boolean]]
+      [:container-style {:optional true} [:maybe :map]]]]]
+   :any])
+
+(def ^:private role-icon
+  {:admin        :i/gavel
+   :member       :i/members
+   :token-master :i/token-master
+   :owner        :i/crown})
 
 (defn- view-internal
-  "Options:
+  [{:keys [actions description description-text error-message role button-one-label button-two-label
+           blur? button-one-props button-two-props theme scroll? container-style]}]
+  [blur/view
+   {:blur-amount   20
+    :blur-type     :transparent
+    :overlay-color (if blur?
+                     colors/neutral-80-opa-1-blur
+                     (colors/theme-colors colors/white-70-blur
+                                          colors/neutral-95-opa-70-blur
+                                          theme))}
```

Comment:
@ajayesivan I think this is the problem. Probably wrapping with blur view should be conditional

Reply:
@OmarBasem Thanks for reporting this. I will look into it.

Reply:
@OmarBasem, I recall that @smohamedjavid addressed this issue in one of his PRs, and I couldn't reproduce this issue in the latest develop. Could you confirm if the problem persists in the latest develop branch? If it does, please specify which screens are still experiencing this issue. Thank you!